core-ui/src/components/Predefined/Create/Namespaces/MemoryQuotas.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { FormItem, FormInput, FormLabel } from 'fundamental-react';
+import * as jp from 'jsonpath';
+
+import { Dropdown } from 'react-shared';
+
+export function MemoryInput({
+  label,
+  propertyPath,
+  container = {},
+  setContainer,
+}) {
+  const units = ['K', 'Ki', 'M', 'Mi', 'G', 'Gi', 'Ti', 'T'];
+  const options = [
+    { key: '', text: 'B' },
+    ...units.map(e => ({
+      key: e,
+      text: e,
+    })),
+  ];
+
+  const value = jp.value(container, propertyPath)?.toString() || '';
+  const numericValue = value.match(/^\d*(\.\d*)?/)[0];
+  const unit = value.replace(numericValue, '');
+  const selectedUnit = units.includes(unit) ? unit : '';
+
+  const setValue = val => {
+    jp.value(container, propertyPath, val);
+    setContainer({ ...container });
+  };
+
+  return (
+    <FormItem>
+      <FormLabel required>{label}</FormLabel>
```

Comment:
You can use the FormField component with `Number` type. Please check the `Issuers` create form as an example